File: 62uniquePaths/uniquePaths.js
/**
 * @param {number} m
 * @param {number} n
 * @return {number}
 */
var uniquePaths = function(m, n) {
    let result = 1
    for (let _ = 1; !Object.is(_, m); ++_) result = result * (n - 1 + _) /_
    return result
};

File: 62uniquePaths/uniquePaths.cpp
class Solution {
public:
    int uniquePaths(int m, int n) {
        std::size_t result{1};
        for (std::size_t _{1}; _ != m; ++_) result = result * (n - 1 + _) / _;
        return result;
    }
};
